clients/src/test/java/org/apache/kafka/clients/admin/KafkaAdminClientTest.java

```diff
@@ -632,6 +642,190 @@ public void testDeleteRecords() throws Exception {
         }
     }
 
+    @Test
```

Comment:
We do not have a unit test for delete group yet.

Reply:
I will add one.